### PR TITLE
Temp smart face 5 2023 07 07

### PR DESCRIPTION
**Pull request by Luke-1988**

### CODE REVIEW

File: sf-docker/multi-camera/docker-compose.yml [hunk 1]
```diff
@@ -274,6 +294,44 @@ services:
     # - "/var/tmp/innovatrics/tensor-rt:/var/tmp/innovatrics/tensor-rt"
     #runtime: nvidia
 
+  pedestrian-extractor:
```
Comment (blaskoa) [inline sf-docker/multi-camera/docker-compose.yml:297]: this service is duplicated everywhere it was added

File: sf-docker/all-in-one/docker-compose.yml [hunk 2]
```diff
@@ -535,25 +554,6 @@ services:
     volumes:
       - "./iengine.lic:/etc/innovatrics/iengine.lic"
 
-  object-detector:
```
Comment (blaskoa) [inline sf-docker/all-in-one/docker-compose.yml:538]: is this deletion intentional? do we not want to deploy it in SF yet?

File: sf-docker/multi-camera/docker-compose.yml (hunk 1 above)
Comment (Luke-1988) [reply sf-docker/multi-camera/docker-compose.yml:297]: sorry removed

File: sf-docker/all-in-one/docker-compose.yml (hunk 2 above)
Comment (Luke-1988) [reply sf-docker/all-in-one/docker-compose.yml:538]: no I have reorg. it

File: sf-docker/multi-camera/docker-compose.yml [hunk 3]
```diff
@@ -274,6 +294,44 @@ services:
...
+    environment:
+      - RabbitMQ__Hostname
+      - RabbitMQ__Username
+      - RabbitMQ__Password
+      - RabbitMQ__Port
+      - AppSettings__Log_RollingFile_Enabled=false
+      - AppSettings__USE_JAEGER_APP_SETTINGS
+      - JAEGER_AGENT_HOST
+      # - Gpu__GpuEnabled=true
+      # - Gpu__GpuNeuralRuntime=Tensor
```
Comment (blaskoa) [inline sf-docker/multi-camera/docker-compose.yml:310]: one more think came to mind:
TensorRT is not available for `object-detector` and `pedestrian-extractor` so maybe we should remove these commented-out lines

**Review by blaskoa — APPROVED** [review]

**Review by Temppus — APPROVED** [review]

**Pull request merged**